tests/testthat/test-fims-estimation.R

```diff
@@ -389,103 +390,134 @@ test_that("estimation test of fims", {
     control = list(maxit = 1000000, reltol = 1e-15)
   ))
 
-  report <- obj$report()
+  report <- obj$report(obj$env$last.par.best)
```

Comment:
Perhaps include documentation here as a reminder for developers to provide `obj$env$last.par.best` to obj$report.